stores/bookstore.ts

```diff
@@ -9,11 +16,74 @@ export const useBookstoreStore = defineStore('bookstore', () => {
     bookstoreInfoByNFTClassIdMap.value[nftClassId] = data
   }
 
+  /* Bookstore CMS */
+
+  const bookstoreCMSTagsByIdMap = ref<Record<string, BookstoreCMSTagItem>>({})
+  const getBookstoreCMSProductsByTagId = computed(() => (tagId: string) => {
+    return bookstoreCMSTagsByIdMap.value[tagId]?.items || []
+  })
+  const getBookstoreCMSProductsByTagIdIsFetching = computed(() => (tagId: string) => {
+    return bookstoreCMSTagsByIdMap.value[tagId]?.isFetching || false
+  })
+  const getBookstoreCMSProductsByTagIdHasFetched = computed(() => (tagId: string) => {
+    return bookstoreCMSTagsByIdMap.value[tagId]?.hasFetched || false
+  })
+  const getBookstoreCMSProductsByTagIdOffset = computed(() => (tagId: string) => {
+    return bookstoreCMSTagsByIdMap.value[tagId]?.offset || undefined
+  })
+
+  async function fetchCMSProductsByTagId(tagId: string, {
+    isMore = false,
```

Comment:
It seems we use append more than refresh, not sure if the optional param should be `isRefresh` instead, and we default to append mode